lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1299,11 +1299,13 @@ export class StandardController implements IController {
         // Block redirectUri opened in a popup from calling MSAL APIs
         BrowserUtils.blockAcquireTokenInPopups();
 
-        // Block token acquisition before initialize has been called if native brokering is enabled
-        BrowserUtils.blockNativeBrokerCalledBeforeInitialized(
-            this.config.system.allowNativeBroker,
-            this.initialized
-        );
+        // Block token acquisition before initialize has been called if native brokering is enabled in top-frame
+        if (setInteractionInProgress) {
```

Comment:
The check has always been `interactionInProgress` to identify an embedded frame I presume?

Reply:
Yes, I just need to change the naming a bit.